dissect/target/plugin.py

```diff
@@ -683,10 +683,14 @@ def _walk(
                             prev_module_path=module_path,
                         )
 
-    yield from _walk(
-        _get_plugins(),
-        special_keys=special_keys,
-        only_special_keys=only_special_keys,
+    yield from sorted(
+        _walk(
+            _get_plugins(),
+            special_keys=special_keys,
+            only_special_keys=only_special_keys,
+        ),
+        key=lambda el: len(el["module"]),
```

Comment:
```suggestion
        key=lambda plugin: len(plugin["module"]),
```
or `plugin_desc` or something

I noticed I didn't add any return type to `_walk()` which doesn't help either.